### PR TITLE
ISPN-6598 Wait for cluster to form before starting the test

### PR DESCRIPTION
**Pull request by slaskawi**
https://issues.jboss.org/browse/ISPN-6598

### CODE REVIEW

File: core/src/test/java/org/infinispan/test/MultipleCacheManagersTest.java [hunk 1]
```diff
@@ -70,6 +70,7 @@ private void callCreateCacheManagers() throws Throwable {
       try {
          log.debug("Creating cache managers");
          createCacheManagers();
+         waitForClusterToForm();
```
Comment (anistor) [inline core/src/test/java/org/infinispan/test/MultipleCacheManagersTest.java]: What's the impact of this on state transfer tests?

Comment (slaskawi) [inline core/src/test/java/org/infinispan/test/MultipleCacheManagersTest.java]: Ouch - quite significant :( Let me put this only the tests which need it.

Comment (slaskawi) [inline core/src/test/java/org/infinispan/test/MultipleCacheManagersTest.java]: Fixed

**Comment by anistor** [comment]
Looks good. Pulling..

**Comment by anistor** [comment]
Integrated in master. Thanks @slaskawi !

**Pull request closed without merging**